src/helpers/NRF52Board.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include <MeshCore.h>
+#include <Arduino.h>
+
+#if defined(NRF52_PLATFORM)

```

Comment:
Same here.

Reply:
Ok, let me try and test the suggestion this weekend.

Should be no issue.

Reply:
You can use what I did in https://github.com/meshcore-dev/MeshCore/pull/1201 if you want. Or maybe just rebase after this was merged.